framework/contrib/hit/Makefile

```diff
@@ -42,13 +46,13 @@ endif
 wasp_CXXFLAGS  += -I$(WASP_DIR)/include
 wasp_LDFLAGS   += -Wl,-rpath,$(WASP_DIR)/lib -L$(WASP_DIR)/lib $(wasp_LIBS)
 
-hit: main.cc parse.cc lex.cc braceexpr.cc braceexpr.h lex.h parse.h
-	$(CXX) -std=c++17 $(wasp_CXXFLAGS) -g $(CXXFLAGS) $< parse.cc lex.cc braceexpr.cc -o $@ $(wasp_LDFLAGS)
+hit: $(hit_srcdir)/main.cc $(hit_srcfiles) include/hit/braceexpr.h include/hit/lex.h include/hit/parse.h
+	$(CXX) -std=c++17 $(wasp_CXXFLAGS) $(hit_includeflags) -g $(CXXFLAGS) $< $(hit_srcfiles) -o $@ $(wasp_LDFLAGS)
```

Comment:
Can you add a `@echo Building hit executable...`  
I think this might be the only build step that echos the compile command instead of a human readable explanation of what's being done.